### DISCUSSION TITLE
Avatar for current user as well

### DISCUSSION THREAD

**Discussion by zzxap** [flyerhq/flutter_chat_ui#658] [Ideas & Requests]
how to show user name and photo on right of chat list ??

only show user name and photo on left.but nothing on right.

**Comment by demchenkoalex** [comment]
On the right it is always logged in user, you can add a header on top of the chat widget with user details, I did not add this on purpose to not repeat the same info on every message.

**Comment by demchenkoalex** [comment]
more details: https://github.com/flyerhq/flutter_chat_ui/issues/643

**Reply by harini18** [reply]
In a group Chat, we like to see who send the message with their Avatar. 
<img width="1206" height="2622" alt="Simulator Screenshot - iPhone 17 Pro - 2026-02-02 at 16 23 17" src="https://github.com/user-attachments/assets/41a77b64-26ba-40de-8b44-96dee7e6fbcd" />

**Reply by harini18** [reply]
How can make this happen using FlutterChatView?

**Reply by demchenkoalex** [reply]
now you control this, check local.dart example - `chatMessageBuilder` and `shouldShowAvatar`. You decide when to show it and how to render it.
